### PR TITLE
add openshift resource inventory metrics to all openshift integrations

### PR DESCRIPTION
**Pull request by maorfr**
follow up on #3858

### CODE REVIEW

**Review by janboll — APPROVED** [review]

File: reconcile/terraform_resources.py [hunk 1]
```diff
@@ -442,6 +442,7 @@ def run(
     # populate the resource inventory with latest output data
     populate_desired_state(ri, resource_specs)
 
+    ob.publish_metrics(ri, QONTRACT_INTEGRATION.replace("_", "-"))
```
Comment (hemslo) [inline reconcile/terraform_resources.py:445]: Why this one has a special replace, is it a requirment that `_` should be `-` in metric name? Should this be handled inside `db.publish_metrics` method?

Comment (maorfr) [reply reconcile/terraform_resources.py]: https://github.com/app-sre/qontract-reconcile/pull/3863/commits/7f125862723bf9e057e4b2d11cfddbc78032152f

**Review by hemslo — APPROVED** [review]
LGTM!

**Review by geoberle — APPROVED** [review]
❤️

**Pull request merged**